blockchain/src/index.js

```diff
@@ -43,6 +43,8 @@ const P2P_PORT = process.env.P2P_PORT || 7447;
 const API_PROTO = process.env.API_PROTO || "http";
 const API_HOST = process.env.API_HOST || "localhost";
 const API_PORT = process.env.API_PORT || "8080";
+const MASTER_API_HOST = process.env.MASTER_API_HOST || API_HOST;
+const MASTER_API_PORT = process.env.MASTER_API_PORT || API_PORT;
```

Comment:
`MASTER_API_HOST || API_HOST`  ensures that if MASTER_API_HOST environment variable is not set, API_HOST is used, and the application behaves the same as before.